### ISSUE TITLE
Turn by turn navigation map doesn't show speed limit UI

### ISSUE THREAD

**Issue by iharshalm** [mapbox/mapbox-navigation-ios#2689]
After starting turn by turn navigation SDK, Map doesn't shows speed limit UI on right corner. 

![Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-12 at 23 00 40](https://user-images.githubusercontent.com/24219374/95774867-e50c6580-0cde-11eb-9b7d-7f44a3a98de1.png)

Below is code -

Direction.shared.calculate(options)  { (session, result) in)  {
switch result {
            case .failure(let error):
                print(error.localizedDescription)
            case .success(let response):
                print("Response->", response)
                guard let route = response.routes?.first else {
                    return
                }
                
                let navigationService = MapboxNavigationService(route: route, routeIndex: 0, routeOptions: routeOptions, simulating: .always)
                let navigationOptions = NavigationOptions(styles: [MichelinDayStyle()], navigationService: navigationService)
                let navigationViewController = NavigationViewController(for: route, routeIndex: 0, routeOptions: routeOptions, navigationOptions: navigationOptions)
                navigationViewController.modalPresentationStyle = .fullScreen
                navigationViewController.showsSpeedLimits = true
                self.present(navigationViewController, animated: true, completion: nil)
     }
}
Mapbox SDK version 6.0

**Comment by 1ec5** [comment]
Does the speed limit indicator appear later on along this route? The screenshot above was taken as the (simulated) user was still on a highway on-ramp. Highway on-ramps in the U.S. have neither legal posted speed limits nor advisory speed limits, so I wouldn’t expect the speed limit indicator to appear until after the user merges onto I-280 (where the white arrow is displayed).

**Comment by iharshalm** [comment]
No, I'm not seeing speed limit data on another remaining route as well. Not sure code mentioned in the ticket/mail/issue is enough for displaying speed limit data.

**Comment by 1ec5** [comment]
There’s a known issue where Valhalla-based Directions API endpoints aren’t returning the metadata required to display the speed limit annotation. As a workaround, if you’re currently using the `.automobile`, `.cycling`, or `.walking` profile identifier, changing the profile identifier to `.automobileAvoidingTraffic` may address the issue for you for now.

**Comment by iharshalm** [comment]
![screenshot](https://user-images.githubusercontent.com/24219374/95876937-3d9c3b00-0d91-11eb-933d-3e5407ea85fb.png)

Tried with automobileAvoidingTraffic but no luck. Also my navigation looking like in above screenshot

**Comment by 1ec5** [comment]
That’s no good. That starting-in-the-ocean is usually indicative of a geometry encoding problem on the server side or, more rarely, a geometry decoding problem on the client side. Let’s follow up on that point in the support ticket you’ve opened, since it might not affect everyone who’s missing the speed limit indicator.

**Comment by iharshalm** [comment]
It's not starting in ocean. Location is correct but just not displaying map/land.
